catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -316,7 +316,7 @@
 
         <AD id="theme" 
             name="Theme" 
-            description="Specifies the default theme.  Custom requires filling out the colors below."
+            description="Specifies the default webapp.  Custom requires filling out the colors below."
```

Comment:
:question: What is inaccurate about saying that this specifies the default theme?

Reply:
I don't think I modified that in this PR.

Reply:
Could you please clarify? This change is present in this pull request.